HelmetCamera/HelmetCamera.cs

```diff
@@ -112,29 +123,38 @@ private IEnumerator LoadSceneEnter()
             // Waiting ~5 seconds for scene is fully loaded
             yield return new WaitForSeconds(5f);
             isCoroutineStarted = true;
-            bool isCameraFound = GameObject.Find("Environment/HangarShip/Cameras/ShipCamera") != null;
+
+            // Reduce amount of searches by caching
+            GameObject foundHangarShip = GameObject.Find("Environment/HangarShip");
+            Transform foundMonitorWall = foundHangarShip.transform.Find("ShipModels2b/MonitorWall");
+            Transform foundShipCamera = foundHangarShip.transform.Find("Cameras/ShipCamera");
+
+            bool isCameraFound = foundShipCamera != null;
             if (isCameraFound)
             {
-                Debug.Log("[HELMET_CAMERAS] Ship camera founded...");
+                Debug.Log("[HELMET_CAMERAS] Ship camera found...");
                 if (!isMonitorChanged)
                 {
-                    GameObject.Find("Environment/HangarShip/ShipModels2b/MonitorWall/Cube")
-                        .GetComponent<MeshRenderer>().materials[2].mainTexture = GameObject.Find("Environment/HangarShip/ShipModels2b/MonitorWall/Cube.001")
+                    Transform foundCube001 = foundMonitorWall.Find("Cube.001");
+                    
+                    foundMonitorWall.Find("Cube")
+                        .GetComponent<MeshRenderer>().materials[2].mainTexture = foundCube001
                         .GetComponent<MeshRenderer>().materials[2].mainTexture;

```

Comment:
the get after set looks rather pointless. Should it be removed?

Reply:
The indentation confused me, so I misinterpreted the meaning of these lines.

Reply:
`"Cube"`'s mainTexture is being set to texture of `"Cube.001"` and later `"Cube.001"`'s mainTexture gets set to custom renderTexture, so no, it shouldn't be removed as it changes execution outcome